examples/orchestrated-apps/pages/orchestrated/_component.vue

```diff
@@ -0,0 +1,22 @@
+<script>
+import FooterComponent from '~/components/FooterComponent.vue'
+import HeaderComponent from '~/components/HeaderComponent.vue'
+
+const componentMap = {
+  footer: FooterComponent,
+  header: HeaderComponent
+}
+export default {
+  layout: 'orchestrated',
+  render (h) {
+    const component = process.server
+      ? this.$route.params.component
+      : this.$root.$options.orchestrated
```

Comment:
This isn't ideal, but client-side we can't rely on the route since it's unrelated to our orchestrated component.  There is probably a cleaner way to do this